docs/dev/table/connectors/formats/canal.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Canal Format"
+nav-title: Canal
+nav-parent_id: sql-formats
+nav-pos: 5
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<span class="label label-info">Changelog-Data-Capture Format</span>
+<span class="label label-info">Format: Deserialization Schema</span>
+
+* This will be replaced by the TOC
+{:toc}
+
+[Canal](https://github.com/alibaba/canal/wiki) is a CDC (Changelog Data Capture) tool that can stream changes in real-time from MySQL into other systems. Canal provides an unified format schema for changelog and supports to serialize messages using JSON and [protobuf](https://developers.google.com/protocol-buffers).
+
+Flink supports to interpret Canal JSON messages as INSERT/UPDATE/DELETE messages into Flink SQL system. This is useful in many cases to leverage this feature, such as synchronizing incremental data from databases to other systems, auditing logs, materialized views on databases, temporal join changing history of a database table and so on.
```

Comment:
Also give a hyperlink for CDC.

Reply:
Needs to illustrate that protobuf is the default format for canal.

Reply:
`such as ... and so on` is too long sentence, can we split and use a list `<ul></ul>` here.

Reply:
What hyperlink do you want to link for CDC?

Reply:
The WIKI seems good to put here: https://en.wikipedia.org/wiki/Change_data_capture

Reply:
I don't think we need to link CDC. The link to Canal and Debezium has a very detailed documentation about what they are and how they do it. The CDC wiki lists many approaches to capture changes, however Canal and Debezium uses one of the approaches.